T04_munsakad_sherrif.py

```diff
@@ -0,0 +1,23 @@
+from time import sleep
+delay = 2
+direction = input("Go with him further [Up, or Down]?")
+if direction == "Up":
+    print("You've reached a large vault... a large console stands to its right...")
+    print("The console lights up when you reach it... it asks for a passcode")
+code = input("Guess the code/Ask for help")
```

Comment:
This line is outside of the `if` condition. I think it should be indented inside it (as well as any subsequent code that is supposed to be inside of the `if direction == "up"` body.